libs/dyn/dynvar/ref.go

```diff
@@ -6,9 +6,7 @@ import (
 	"github.com/databricks/cli/libs/dyn"
 )
 
-const VariableRegex = `\$\{([a-zA-Z]+([-_]?[a-zA-Z0-9]+)*(\.[a-zA-Z]+([-_]?[a-zA-Z0-9]+)*(\[[0-9]+\])*)*(\[[0-9]+\])*)\}`
-
-var re = regexp.MustCompile(VariableRegex)
+var re = regexp.MustCompile(`\$\{([a-zA-Z]+([-_]?[a-zA-Z0-9]+)*(\.[a-zA-Z]+([-_]?[a-zA-Z0-9]+)*(\[[0-9]+\])*)*(\[[0-9]+\])*)\}`)
```

Comment:
This was done in https://github.com/databricks/cli/pull/1398 as a bug fix. Since it's no longer used so I undid the change. 

Also, the variable name `VariableRegex` is inaccurate. It really should be something like `ReferenceRegex`.